flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/source/config/PostgresSourceConfigFactory.java

```diff
@@ -88,6 +95,8 @@ public PostgresSourceConfig create(int subtaskId) {
             props.putAll(dbzProperties);
         }
 
+        props.put("snapshot.mode", "never");
```

Comment:
pls add a note like:
The  PostgresSource will do snapshot according to the its StartupMode, do not need debezium do the snapshot work